test/yaml_writer_test.rb

```diff
@@ -23,34 +23,38 @@
 require_relative "spec_helper"
 require "storage"
 require "y2storage/yaml_writer"
+require "y2storage/disk"
+require "y2storage/partition_tables"
+require "y2storage/filesystems"
 
 describe Y2Storage::YamlWriter do
 
-  it "produces yaml of a simple disk and partition setup" do
+  let(:test_environment) do
+    Storage::Environment.new(true, Storage::ProbeMode_NONE, Storage::TargetMode_DIRECT)
+  end
+  let(:storage) { Storage::Storage.new(test_environment) }
```

Comment:
Well, this change is in fact using the new wrapper in the RSpec test, but it does not adapt the YamlWriter itself (neither proves it works without needing adaptation, if that's the case). Is that planned for an upcoming pull request?

I mean, doing something like:
```ruby
describe Y2Storage::YamlWriter do
  before do
    Y2Storage::StorageManager.create_test_instance
  end
  # Using y2storage_staging is temporary to not break compatibility with stuff accessing libstorage directly
  let(:staging) { Y2Storage::StorageManager.instance.y2storage_staging }

  it "produces yaml of a simple disk and partition setup" do
     [...]
     Y2Storage::YamlWriter.write(staging, io)
  end
end
```

Because after this pull request, YamlWriter still expects a `Storage::Devicegraph` object instead of a `Y2Storage::Devicegraph` one.